go.mod

```diff
@@ -9,10 +9,10 @@ require (
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/covalent-indexer-go v1.0.6
 	github.com/ElrondNetwork/elastic-indexer-go v1.1.32
-	github.com/ElrondNetwork/elrond-go-core v1.1.7
+	github.com/ElrondNetwork/elrond-go-core v1.1.8-0.20220112192542-db36097f7385
```

Comment:
do not forget about proper releases and after that `go mod tidy`

Reply:
done